go/store/nbs/table_index_test.go

```diff
@@ -229,25 +231,24 @@ func TestAmbiguousShortHash(t *testing.T) {
 
 // fakeChunk is chunk with a faked address
 type fakeChunk struct {
-	address addr
+	address hash.Hash
 	data    []byte
 }
 
 var fakeData = []byte("supercalifragilisticexpialidocious")
 
-func addrFromPrefix(prefix string) (a addr) {
+func addrFromPrefix(prefix string) hash.Hash {
```

Comment:
Nit: This function looks super busted to me. Unused or something? An encoded hash should be StringLen, not ByteLen, and this function doesn't even pad the passed prefix out to ByteLen...

Reply:
fixed